### DISCUSSION TITLE
How to bind pause/break key?

### DISCUSSION THREAD

**Discussion by liamjones** [Hammerspoon/hammerspoon#2905] [Q&A] [answered]
Is it currently impossible to bind something to the pause/break key in Hammerspoon? I don't see it listed here (unless I'm missing a non-obvious name): https://www.hammerspoon.org/docs/hs.keycodes.html#map

**Comment by royathan** [comment, marked as answer]
The **Pause** key on my keyboard (Logitech G915) emits F15 when pressed on MacOS.

So the following works for me:
```lua
hs.hotkey.bind(nil, "F15", function()
  hs.alert.show("Hello World!")
end)
```

**Reply by liamjones** [reply]
Ah! That works, thanks!

**Comment by NightMachinery** [comment]
See https://github.com/Hammerspoon/hammerspoon/issues/3350 .
